release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4040,4 +4040,23 @@ $ oc adm release info 4.12.32 --pullspecs
 [id="ocp-4-12-32-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-33"]
+=== RHBA-2023:5016 - {product-title} 4.12.33 bug fix update
+
+Issued: 2023-09-12
+
+{product-title} release 4.12.33 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5016[RHBA-2023:5016] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5018[RHBA-2023:5018] advisory.
```

Comment:
This URL gives me a 404 not found error (I am connected to the VPN and can view other advisories).

Reply:
Yeah, these links won't be available on the Customer Portal until the release date, which is the 12th of September. At the moment, the content is on Errata advisories. The peer review usually occurs in advance of the release day, so we can coordinate the docs release with the z-stream release binaries. We ensure we do not merge the PR until the binary release occurs, so the onus on the links working falls on the merge phase.